content/attacks/posts/2023-11-10-RaftExploit

```diff
@@ -0,0 +1,31 @@
+---
+date: 2023-11-10  
+target-entities: Raft R
+entity-types: DeFi
+Attack-types: Exploit
+title: “Raft’s exploited by $3.3M”
+---
+
+## Summary
+
+Raft R, a decentralized finance (DeFi) platform, suffered a $3.3 million exploit that resulted in a 50% drop in the value of its R [stablecoin.](https://www.coindesk.com/tech/2023/11/10/defi-platform-raft-suffers-33m-exploit-but-hacker-likely-takes-a-loss-on-the-attack/#:~:text=stablecoin%20dropped%20as-,much%20as%2050%25,-from%20its%20supposed) The attacker drained 1,577 ETH from Raft by minting R tokens. The rcbRTH-c index of the collateral token was manipulated thus creating $6.7 million of R [tokens](https://twitter.com/PeckShieldAlert/status/1723135066522702258) before he [sold](https://twitter.com/davgarai/status/1723065357445775507?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E1723065357445775507%7Ctwgr%5E62cf35944c90932f69e23dc8f627c581d62e6ffa%7Ctwcon%5Es1_&ref_url=https%3A%2F%2Fcoincu.com%2F227678-raft-hack-led-to-r-minted%2F) them to drain liquidity, and withdrawal the collateral. The attacker then sent 1,570 ETH to a burn [address](https://platform.arkhamintelligence.com/explorer/address/0xc1f2b71A502B551a65Eee9C96318aFdD5fd439fA), leaving only 7 ETH for themselves. Raft's stablecoin initially dropped to 50 percent but later rebounded to around 70 [cents.](https://www.dimsumdaily.hk/raft-defi-platform-loses-us3-3-million-in-ether-hack-attacker-faces-loss/#:~:text=later%20recovered%20to-,around%2070%20cents,-%2C%20according%20to%20Coinmarketcap) Raft is working to compensate users using protocol-owned sDAI in the Peg [Stability](https://www.dimsumdaily.hk/raft-defi-platform-loses-us3-3-million-in-ether-hack-attacker-faces-loss/#:~:text=people%20whole%20using-,the%20protocol%2Downed,-sDAI%20in%20the) Module.
+
+## Attackers
+
+The identity of the attacker is currently unknown.
+
+## Losses
+
+Raft R lost 1,577 [ETH](https://blockchain.news/news/defi-platform-raft-compromisedloses-33-million-in-ether#:~:text=resulted%20in%20the%20theft%20of%20around%20%243.3%20million%20worth%20of%20Ether%20(ETH)) during the attack, worth 3,300,000 USD at the time of the attack.
+
+## Timeline
+
+-**November 10, 2023, 14:49 PM:** Raft co-funder David Garai confirmed in a post on X (formerly Twitter) that the platform was targeted by an [attack](https://twitter.com/davgarai/status/1723065357445775507?s=20). 
```

Comment:
sorry, just make sure to convert times to UTC and then add "UTC"

Reply:
Will do. 

Is there a simple way to figure out what time zone an article is posted? A lot of the time it'll just say something like "12 hours ago".

Reply:
depends on how they timestamp it